js/user-form.js

```diff
@@ -0,0 +1,109 @@
+const form = document.getElementById('form');
+const username = document.getElementById('username');
+const first_name = document.getElementById('first_name');
+const last_name = document.getElementById('last_name');
+const yoe = document.getElementById('yoe');
+const company = document.getElementById('company');
+const designation = document.getElementById('designation');
+const github_id = document.getElementById('github_id');
+const linkedin_id = document.getElementById('linkedin_id');
+const twitter_id = document.getElementById('twitter_id');
+const btn = document.getElementById('submission');
+
+//Error Handler
+function showError(input, message) {
+  const formGroup = input.parentElement;
+  formGroup.className = 'form-group error';
+  const small = formGroup.querySelector('small');
+  small.innerText = message;
+}
+
+//Success Handler
+function showSuccess(input) {
+  const formGroup = input.parentElement;
+  formGroup.className = 'form-group success';
+}
+
+// Username Validator
+function checkUsername(input) {
+  //Check presence of first_name
+  if (!username.value.trim().startsWith(first_name.value.toLowerCase())) {
+    showError(
+      username,
+      'Username should start with your first name. Please avoid using funky words as suffix'
```

Comment:
No, @swarajpure ! I believe the username term would be coined starting from the respective user's firstName and then optionally a hyphen or alphabetical suffixes.

Reply:
What I see from the code is that the user  is not allowed to use a username that does NOT start with their first name. So the error should show that the user should not have a username which doesn't START with their first name and hence I was asking if you meant to use `prefix` here instead of `suffix`.